services/frontend-v3/src/components/careerInterests/CareerInterestsView.jsx

```diff
@@ -11,15 +11,18 @@ const CareerInterestsView = ({ info, relocationLocationsInfo }) => {
         dataSource={dataSource}
         renderItem={(item) => (
           <List.Item>
-            <List.Item.Meta title={item.title} description={item.description} />
+            <List.Item.Meta
+              title={item.title || undefined}
+              description={item.description || undefined}
+            />
```

Comment:
I would just revert to how it was, since if item.title is undefined, the value of title will be undefined, we don't need to specify undefined
```suggestion
            <List.Item.Meta
              title={item.title}
              description={item.description}
            />
```